utils/casctl

```diff
@@ -126,7 +126,7 @@ class cas:
         parser_stop.set_defaults(command='stop')
         parser_stop.add_argument ('--flush', action='store_true', help = 'Flush data before stopping')
 
-        args = parser.parse_args(sys.argv)
+        args = parser.parse_args(sys.argv[1::])
```

Comment:
So this commit actually reverts change from the first commit. This second colon doesn't seem to be necessary.

It seems that following commits of your patchset fix thighs broken by the first one. This is highly unrecommended. The rule is "if you break something and then fix it in the same patchset, do this in the same commit". Please squash the commits into one.

Reply:
done